packages/x-charts/src/context/CartesianProvider/computeValue.ts

```diff
@@ -27,6 +27,31 @@ const getRange = (drawingArea: DrawingArea, axisName: 'x' | 'y', isReverse?: boo
   return isReverse ? range.reverse() : range;
 };
 
+const zoomedScaleRange = (scaleRange: [number, number] | number[], zoomRange: [number, number]) => {
+  const rangeGap = scaleRange[1] - scaleRange[0];
+  const rangeMinMultiplier = zoomRange[0] / 100;
+  const rangeMaxMultiplier = zoomRange[1] / 100;
+  const min = (scaleRange[0] - rangeGap * rangeMinMultiplier) / (1 - rangeMinMultiplier);
+  const max = (scaleRange[1] + rangeGap * (1 - rangeMaxMultiplier)) / rangeMaxMultiplier;
```

Comment:
@alexfauquette this math is wrong and I can't pinpoint exactly why.

The idea here is that we are zooming in on the scale range `[svg coords]` based on the zoomRange, `[0,100]`. I'm trying to transform the scale so the proportions are kept, but I'm doing something wrong there.

The scales pct sum should be 1, but they are going above 1 depending on where in the chart you pan to. Eg: zoom in, if you pan to the start < of the graph, the pct is ~1, but to the other side >, it gets to almost 2 😅